src/main/java/de/komoot/photon/metrics/MetricsConfig.java

```diff
@@ -0,0 +1,85 @@
+package de.komoot.photon.metrics;
+
+import de.komoot.photon.CommandLineArgs;
+import io.javalin.micrometer.MicrometerPlugin;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.jvm.ClassLoaderMetrics;
+import io.micrometer.core.instrument.binder.jvm.JvmGcMetrics;
+import io.micrometer.core.instrument.binder.jvm.JvmMemoryMetrics;
+import io.micrometer.core.instrument.binder.jvm.JvmThreadMetrics;
+import io.micrometer.core.instrument.binder.system.DiskSpaceMetrics;
+import io.micrometer.core.instrument.binder.system.FileDescriptorMetrics;
+import io.micrometer.core.instrument.binder.system.ProcessorMetrics;
+import io.micrometer.core.instrument.binder.system.UptimeMetrics;
+import io.micrometer.prometheusmetrics.PrometheusConfig;
+import io.micrometer.prometheusmetrics.PrometheusMeterRegistry;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+
+public class MetricsConfig {
+    private static final Logger LOGGER = LogManager.getLogger();
+    private MicrometerPlugin micrometerPlugin;
+    private PrometheusMeterRegistry registry;
+    private String path = "/metrics";
+
+    private MetricsConfig() {}
+
+    private void init(String metricsPath) {
+        registry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
+        registry.config().commonTags("application", "Photon");
+        registerJvmMetrics(registry);
+        if (metricsPath != null && !metricsPath.isEmpty()) {
+            path = metricsPath;
+        }
+        micrometerPlugin = new MicrometerPlugin(micrometerPluginConfig -> micrometerPluginConfig.registry = registry);
+        LOGGER.info("Metrics enabled at " + path);
+    }
+
+    private void registerJvmMetrics(MeterRegistry registry) {
+        new ClassLoaderMetrics().bindTo(registry);
+        new JvmMemoryMetrics().bindTo(registry);
+        new JvmGcMetrics().bindTo(registry);
+        new JvmThreadMetrics().bindTo(registry);
+        new UptimeMetrics().bindTo(registry);
+        new ProcessorMetrics().bindTo(registry);
+        new DiskSpaceMetrics(new File(System.getProperty("user.dir"))).bindTo(registry);
+        new FileDescriptorMetrics().bindTo(registry);
```

Comment:
The latter four look like system-wide metrics. Seems odd to have them exported by Photon.

The current size of the database could be an additional interesting data point, though. It doesn't matter when using a static import but becomes relevant when running updates.